amivapi/tests/events/test_queue.py

```diff
@@ -59,6 +57,23 @@ def test_fcfs_users_get_auto_accepted(self):
                                status_code=200).json['_items']
         self.assertEqual(len(waiting), 0)
 
+    def test_fcfs_users_get_auto_accepted_unlimited_spots(self):
+        """Test that with fcfs the users get automatically accepted on signup
+        for events with unlimited spaces"""
+        event = self.new_object('events', spots=0,
+                                selection_strategy='fcfs')
+
+        user = self.new_object('users')
+
+        self.api.post('/eventsignups', data={
+            'user': str(user['_id']),
+            'event': str(event['_id'])
+        }, status_code=201)
+
+        accepted = self.api.get('/eventsignups?where={"accepted":true}',
+                                status_code=200).json['_items']
+        self.assertEqual(len(accepted), 1)
```

Comment:
I think it would be more straightforward to check for the particular signup we created and check if *this* signup is accepted, instead of checking if there is one accepted signup in total.

Reply:
I will quickly add this.